commit b6ccf1a0874333cf20ab9faa3cfb4c1d73850b04
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-08-22T14:28:41-07:00 (7 weeks ago)
    
    joyent/triton-dehydrated#20 dehydrated dependency sometimes mis-handles account ID

diff --git a/deps/dehydrated b/deps/dehydrated
index fba49ba..0182549 160000
--- a/deps/dehydrated
+++ b/deps/dehydrated
@@ -1 +1 @@
-Subproject commit fba49ba28eb746eae127aa36e7f515beeaf0bbae
+Subproject commit 018254974cb6cbf93ba9249b06b2d923e37a9e88
